src/Umbraco.Web.UI.Client/src/packages/documents/documents/workspace/document-workspace-split-view.element.ts

```diff
@@ -35,6 +39,13 @@ export class UmbDocumentWorkspaceSplitViewElement extends UmbLitElement {
 		);
 	}
 
+	#observeIcon() {
+		if (!this._workspaceContext) return;
```

Comment:
Its more ideal to have the this.observe method called with undefined in the case of the context begin gone. Because this will stop the old observation and then reset the icon to undefined. as the observation callback will get the value of undefined.

Reply:
It is fixed 👍